kubectl-rook-ceph.sh

```diff
@@ -27,21 +27,23 @@ function print_usage () {
   echo "USAGE"
   echo "  kubectl rook-ceph <main args> <command> <command args>"
   echo "MAIN ARGS"
-  echo "  -h, --help                  : output help text"
-  echo "  -n, --namespace='rook-ceph' : the namespace of the CephCluster"
+  echo "  -h, --help                                : output help text"
+  echo "  -n, --namespace='rook-ceph'               : the namespace of the CephCluster"
+  echo "  -op-ns, --operator-namespace='rook-ceph'  : the namespace of the CephCluster"
+
   echo "COMMANDS"
-  echo "  help                        : output help text"
-  echo "  ceph <args>                 : call a 'ceph' CLI command with arbitrary args"
-  echo " rbd <args>                   : call a 'rbd' CLI command with arbitrary args"
+  echo "  help                                      : output help text"
```

Comment:
Since `--help` is above, seems like we don't need to document it here as well, even if it's an option.
```suggestion
```

Reply:
removed